src/gtk/toga_gtk/widgets/numberinput.py

```diff
@@ -61,10 +61,13 @@ def set_font(self, font):
         self.interface.factory.not_implemented('NumberInput.set_font()')
 
     def rehint(self):
+        # Does not use self.interface.MIN_WIDTH as a minimum width; because
+        # the min width of the widget will fail when number input width be
+        # larger than self.interface.MIN_WIDTH
```

Comment:
Yes, good and then we need to inform the `Toga` end user with that the minimum width of the `Widget` may be bigger than `MIN_WIDTH` to prevent the confusing, is this correct? Since the end user has an access to `MIN_WIDTH`.

Reply:
Why? Minimum is, by definition, a minimum. Of course it can be bigger. The purpose of MIN_WIDTH is to ensure that you can't end up with nonsense "3 pixel wide" input fields. If the minimum size suggested by the GUI toolkit is bigger than `MIN_WIDTH`, it's presumably for a good visual reason. This is primarily useful for widgets like Canvas, where reducing to 0x0 size doesn't make any sense, but it's a useful safety catch for other widgets.

And yes, `MIN_WIDTH` is externally visible; it's arguable that `MIN_WIDTH` should be `_MIN_WIDTH`, since it won't actually be useful to the end user.

Reply:
As you wrote, I expect notifying the user with following will be good:
If the minimum size suggested by the GUI toolkit is bigger than MIN_WIDTH, it's presumably for a good visual reason. So, we will use it as the minimum size.